lib/guest_installation_and_configuration_base.pm

```diff
@@ -343,7 +343,7 @@ sub prepare_common_environment {
         virt_autotest::utils::setup_common_ssh_config('/root/.ssh/config');
         script_run("sed -irn \'s/^.*IdentityFile.*\$/#&/\' /etc/ssh/ssh_config");
         enable_debug_logging;
-        virt_autotest::utils::setup_rsyslog_host($common_log_folder);
+        virt_autotest::utils::setup_rsyslog_host($common_log_folder) unless is_opensuse;
```

Comment:
You'd better modify inside virt_autotest::utils::setup_rsyslog_host instead of guest_installation_and_configuration_base.pm

Reply:
Have not began to look into this rsyslog problem, just comented it out to continue test. I'll take your point into account in my next TW pr. thanks.

Reply:
Yes. Please. It is very easy to do. Just replace zypper_call with script_run("zypper automatic install rsyslog").

Reply:
> the way: script_run("zypper automatic install rsyslog")

`"# zypper automatic install rsyslog
Unknown command 'automatic'"`

> - Or you can use zypper_call with exitcode => xxx option to allow failure.

'zypper in rsyslog' fails and rsyslog is not installed, as the result the entire function of setup_rsyslog_host() is unable to work and meaningless, why do we have to keep it in openSUSE TW test?